yb-voyager/cmd/exportData.go

```diff
@@ -168,6 +180,106 @@ func exportDataOffline() bool {
 	return true
 }
 
+func debeziumExportData(ctx context.Context, tableList []*sqlname.SourceName) error {
+	absExportDir, err := filepath.Abs(exportDir)
+	if err != nil {
+		return fmt.Errorf("failed to get absolute path for export dir: %v", err)
+	}
+
+	var snapshotMode string
+	if liveMigration {
+		snapshotMode = "initial"
+	} else if useDebezium {
+		snapshotMode = "initial_only"
+	}
+
+	config := &dbzm.Config{
+		SourceDBType: source.DBType,
+		ExportDir:    absExportDir,
+		Host:         source.Host,
+		Port:         source.Port,
+		Username:     source.User,
+		Password:     source.Password,
+
+		DatabaseName: source.DBName,
+		SchemaNames:  source.Schema,
+		TableList:    sqlname.GetMinQuotedTargetTableList(tableList),
+		SnapshotMode: snapshotMode,
+	}
+	debezium := dbzm.NewDebezium(config)
+	err = debezium.Start()
+	if err != nil {
+		return fmt.Errorf("failed to start debezium: %v", err)
+	}
+	var status *dbzm.ExportStatus
+	exportingSnapshot := true
+	for exportingSnapshot {
+		status, err = debezium.GetExportStatus()
+		if err != nil {
+			return fmt.Errorf("failed to read toc: %v", err)
+		}
+		if status != nil {
+			log.Infof("Debezium status: %s", (status.Mode))
+
+		}
+		if status != nil && exportingSnapshot && status.SnapshotExportIsComplete() {
+			exportingSnapshot = false
+			utils.PrintAndLog("Snapshot export is complete.")
+			createExportDataDoneFlag()
+			err = writeDataFileDescriptor(exportDir, status)
+			if err != nil {
+				return fmt.Errorf("failed to write data file descriptor: %v", err)
+			}
+			outputExportStatus(status)
+		}
+		time.Sleep(time.Second)
+	}
+	if err := debezium.Error(); err != nil {
+		return fmt.Errorf("debezium failed during initial snapshot phase: %v", err)
+	}
+
+	if !liveMigration && useDebezium {
```

Comment:
We can assume that exactly one of them is true. So you can only check `if useDebezium`